src/index.js

```diff
@@ -0,0 +1,89 @@
+import { editTodo, removeTodo } from './edit-remove.js';
+import { getFromLocal, updateLocalStorage } from './local-storage';// eslint-disable-line
```

Comment:
- [ ] Please kindly do not disable linters, fix linters errors helps programmers to write clean and clear codebase 😁 😉